src/node_contextify.cc

```diff
@@ -358,19 +336,17 @@ class ContextifyContext {
     if (ctx->context_.IsEmpty())
       return;
 
-    Local<Object> sandbox = PersistentToLocal(isolate, ctx->sandbox_);
     MaybeLocal<Value> maybe_rv =
-        sandbox->GetRealNamedProperty(ctx->context(), property);
+        ctx->sandbox()->GetRealNamedProperty(ctx->context(), property);
```

Comment:
Consider caching `sandbox` and `context` in a local.  Every call adds a new slot to the HandleScope and seeing how this function can get called a lot, efficiency is important.